lib/model.js

```diff
@@ -942,6 +942,10 @@ Connection.from = function(url) {
       attrs.mongodb_database_name = decodeURIComponent(
         parsed.db_options.authSource || parsed.dbName);
     }
+    if (Connection.isAtlas(url) && attrs.mongodb_password === 'PASSWORD') {
```

Comment:
Can we split this into two separate checks?

something like: 

```
if (Connection.isAtlas(url)) {
  attrs.ssl = 'UNVALIDATED';
  if (attrs.mongodb_password === 'PASSWORD') {
    attrs.mongodb_password = '';
  }
}
```

That way, the whole feature doesn't break if the user has the correct password in their URL (not copy&pasted from the Atlas UI). Or if Cloud decides to change the string `PASSWORD` to `MYPASSWORD` in their UI.